### PR TITLE
feat: add move messages to junk folder

### PR DESCRIPTION
**Pull request by kesselb**
### ☑️ Resolves

* Fix https://github.com/nextcloud/documentation/issues/10958

### 🖼️ Screenshots

Screenshot from 2023-08-07 14-42-07

### CODE REVIEW

File: user_manual/groupware/mail.rst [hunk 1]
```diff
@@ -51,6 +51,21 @@ Your account settings such as:
...
 
+Move messages to Junk folder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   .. versionadded:: 3.4
+
+When a message is marked as Junk, Mail can automatically move the message to a different folder.
+
+1) Open Account settings
+2) Goto Default folders
```
Comment (GretaD) [inline user_manual/groupware/mail.rst:62]: space :)

**Review by GretaD — APPROVED** [review]

**Comment by kesselb** [comment]
@ChristophWurst should we backport to 27 and 26?

**Comment by ChristophWurst** [comment]
It's a feature. We can leave it for the next minor only 👍

**Comment by ChristophWurst** [comment]
Scratch that. Yes, please backport :)

**Pull request merged**